src/main/resources/resources/logisim/strings/std/std_de.properties

```diff
@@ -813,6 +813,7 @@ TTL7454 = 7454: Vierfach UND-OR-INVERT-Gatter
 TTL74541 = 74541: Achtfach Puffer / Leitungstreiber
 TTL7458 = 7458: Zweifach AND-OR Gatter
 TTL7464 = 7464: 4-2-3-2-2 AND-OR-INVERT-Gatter
+# ==> TTL74670 =
```

Comment:
4-mal-4-Bit-Registerspeicher mit Tri-State-Ausgängen

Reply:
Done. Thanks once again for your translation